StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Extensions/Intent+Link.swift

```diff
@@ -9,16 +9,16 @@
 @_spi(STP) import StripePayments
 
 extension Intent {
-    var supportsLink: Bool {
-        return recommendedPaymentMethodTypes.contains(.link)
+    func supportsLink(allowV2Features: Bool) -> Bool {
+        return recommendedPaymentMethodTypes.contains(.link) || (linkPassthroughModeEnabled && allowV2Features)
```

Comment:
This seems odd to me -- i would expect that `.link` would need to be contained within  `recommendedPaymentMethodTypes` ?

Reply:
It's also possible that `supportsLink` is true even if Link is not a recommended PaymentMethod type, because in passthrough mode Link acts as a Card PaymentMethod.

Reply:
Added a comment to clarify